demos/notebooks/demo_dendritic.ipynb

```diff
@@ -164,11 +201,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#%% start a cluster for parallel processing (if a cluster already exists it will be closed and a new session will be opened)\n",
-    "if 'dview' in locals():\n",
+    "print(f\"You have {psutil.cpu_count()} CPUs available in your current environment\")\n",
+    "processors_to_use = psutil.cpu_count() - 4 #  None  # Set to number less than cpu_count - 1 if you have memory problems"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#start a cluster for parallel processing \n",
+    "# note if a cluster already exists it will be closed so a new session will be opened\n",
+    "if 'dview' in locals():  # locals contains list of current local variables\n",
+    "    print('Closing previous cluster')\n",
     "    cm.stop_server(dview=dview)\n",
-    "c, dview, n_processes = cm.cluster.setup_cluster(\n",
-    "    backend='local', n_processes=None, single_thread=False)"
+    "print(\"Setting up new cluster\")\n",
+    "c, dview, n_processes = cm.cluster.setup_cluster(backend='local', \n",
```

Comment:
prefer to call this "multiprocessing" in new code; local is one of those older terms we're hoping to phase out